### PR TITLE
button text and russian

### PR DESCRIPTION
**Pull request by JonasJurczok**
I fixed the problem with the button text and added the russian translations.

@Tarrke @hoylemd @gsuberland please take a look :)

### CODE REVIEW

**Comment by JonasJurczok** [comment]
@Tarrke @hoylemd I'm merging this to fix the issues people have with the current version. I would still like to hear your feedback though :)

File: src/todo/todo.lua [hunk 1]
```diff
@@ -147,7 +147,9 @@ function todo.update_current_task_label(player)
         if task.assignee == player.name then
             todo.log(serpent.block(task))
             todo.get_maximize_button(player).caption =
-                {"todo.todo_maximize_button_caption", {"todo.todo_list"}, task.task}
+                {"todo.todo_maximize_button_caption",
+                 {"todo.todo_list"},
+                 string.match(task.task, "[^\r\n]+")}
```
Comment (Tarrke) [inline src/todo/todo.lua]: This should not work with an empty first line. Can be fixed with "[^\r\n]*" instead.
Should be tested as I'm not sure of the lua regex comportment.

Comment (Tarrke) [inline src/todo/todo.lua]: This should not work with an empty first line. Can be fixed with "[^\r\n]*" instead.
Should be tested as I'm not sure of the lua regex comportment.

File: src/todo/todo.lua [hunk 2]
```diff
@@ -187,17 +189,16 @@ end
...
     todo.log("Marking task [" .. id .. "] as completed.")
-    local t
     for i, task in ipairs(global.todo.open) do
         if (task.id == id) then
-            t = table.remove(global.todo.open, i)
+            local t = table.remove(global.todo.open, i)
             todo.log("Removed task from open list.")
+
+            todo.log("Adding task [" .. t.id .. "] to done list.")
+            table.insert(global.todo.done, t)
```
Comment (Tarrke) [inline src/todo/todo.lua]: Good thing done here.

**Comment by Tarrke** [comment]
Sorry for last week, I wasn't there to read mails :).

**Comment by JonasJurczok** [comment]
> Sorry for last week, I wasn't there to read mails :).

No worries. I'm the one who usually needs months to respond.. I won't complain about a week :D

File: src/todo/todo.lua (hunk 1 above)
Comment (JonasJurczok) [reply src/todo/todo.lua]: #67 :)

File: src/todo/todo.lua (hunk 2 above)
Comment (JonasJurczok) [reply src/todo/todo.lua]: Yeah.. I found that, did a git blame, found my name and was like "what the fuck was I thinking?" :D

**Pull request merged**